.github/workflows/ci.yml

```diff
@@ -20,14 +20,14 @@ jobs:
     - name: Configure docker
       run: sudo ./ci/configure-docker.py
     - name: Pull container
-      run: docker pull quay.io/ovirt/imageio-test-${{matrix.distro}}
+      run: docker pull quay.io/alesgar/imageio-test-${{matrix.distro}}
```

Comment:
This is temporary change for testing? The PR should not include these changes. You can
make this in a private branch in your fork for testing, and share a link to the build using these changes, but please avoid sending temporary changes to PRs.

Reply:
The idea was to test it and then remove the commit and leave it ready. But it failed late last Friday, so I left it in.

I'll separate the commit to my fork, and take it into account for future image updates.